runtime/pack/dist/opt/termdebug/plugin/termdebug.vim

```diff
@@ -655,16 +667,16 @@ def StartDebug_prompt(dict: dict<any>)
 
   ch_log($'executing "{join(gdb_cmd)}"')
   gdbjob = job_start(gdb_cmd, {
-    exit_cb: function('EndDebug'),
-    out_cb: function('GdbOutCallback'),
+    exit_cb: EndDebug,
+    out_cb: GdbOutCallback
   })
   if job_status(gdbjob) != "run"
     Echoerr('Failed to start gdb')
     exe $'bwipe! {promptbufnr}'
     return
   endif
   exe $'au BufUnload <buffer={promptbufnr}> ++once ' ..
-       'call job_stop(gdbjob, ''kill'')'
+    'job_stop(gdbjob, ''kill'')'
```

Comment:
I am not sure this is correct. Are you sure the exec'ed string is evaluated in vim9 script semantics?

Reply:
Not really... I replaced it back.